### DISCUSSION TITLE
<tmax> not extending simulation time in pipeflow example

### DISCUSSION THREAD

**Discussion by cerice1995** [UvaCsl/HemoCell#58] [General discussion]
I edited tmax in config.xml while leaving dt, tcsv, etc unchanged. The simulation continues to run for the default <tmax> of 10,000 time steps or .001 s 

Is there somewhere else you need to edit to extend the runtime of the pipeflow example?

**Comment by gzavo** [comment]
Hi @cerice1995,

Indeed, it should be set by `tmax` in the config xml.
We have not encountered this problem so far, so as a first step please double check for trivial errors (e.g., that you really run with the xml that you edit).
If you still have the same problem let us know here and also tell the version that you are using.

**Reply by cerice1995** [reply]
I am using version 2.6 and have rerun checking for trivial errors with the same result.

**Reply by cerice1995** [reply]
It seems that the logfile in the output folder stops after 10,000 steps but if I print the slurm output the code is still running.

**Reply by cerice1995** [reply]
It was resolved by assigning more memory after checking the CPU usage (half the cores were running at 100% and half were sitting idle).
